js/applesoft/tokens.ts

```diff
@@ -0,0 +1,222 @@
+
+/** Map from token to keyword */
+export const TOKEN_TO_STRING = {
```

Comment:
Typing these as `Record<byte, string>` and `Record<string, byte` will avoid having to cast later on.

Reply:
I was trying to treat the map as a type, hence the `as const` at the end. I don't think that extra typing is buying anything, so I'll remove it and see what happens.